Yoast/YoastSEO.js
# 1808 add related keywords

## Summary

This PR can be summarized in the following changelog entry:

* Adds specific SEO assessors for related keywords, separately for regular posts, taxonomy pages, and cornerstone posts.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Start and login to WordPress.
* Start and activate the Premium plugin.

* Make or open a new post. 
* Make sure that...
   * .. the post contains >100 words, to trigger the keyword density assessment.
   * .. the post contains a meta description, to trigger the meta description keyword assessment.
   * .. the post contains an image, to trigger the image alt-text assessment.
* Add a related keyword.
* Check if these assessments, and only these assessments, are run for the related keyword:
   * Keyword density.
   * Introduction keyword.
   * Key phrase length.
   * Image in alt-text.
   * Meta description keyword.
   * Competing links assessment.
* Check if adding multiple related keywords does not break anything.
* Check if the SEO analysis for the main keyword still works as expected.

* Follow the same steps above for a post marked as a cornerstone article.
* This time, check if these and only these assessments are run for the related keyword:
   * Keyword density.
   * Introduction keyword.
   * Key phrase length.
   * Image in alt-text.
      * Make sure that this analysis uses the cornerstone scores and boundaries.
   * Meta description keyword.
   * Competing links assessment.


* Follow the same steps above for a category page.
* This time, check if these and only these assessments are run for the related keyword:
   * Keyword density.
   * Introduction keyword.
   * Key phrase length.
   * Meta description keyword.


Fixes #1808
